crates/sail-spark-connect/tests/gold_data/plan/ddl_create_table.json

```diff
@@ -501,7 +957,53 @@
     {
       "input": "CREATE TABLE my_tab(a INT comment 'test', b STRING) USING parquet PARTITIONED BY (a)",
       "output": {
-        "failure": "invalid argument: sql parser error: Expected: a data type name, found: ) at Line: 1, Column: 84"
+        "success": {
+          "command": {
+            "createTable": {
+              "table": [
+                "my_tab"
+              ],
+              "schema": {
+                "fields": [
+                  {
+                    "name": "a",
+                    "dataType": "integer",
+                    "nullable": true,
+                    "metadata": []
```

Comment:
It seems the column comment is not parsed. Maybe we can add a TODO somewhere in the code.